src/scenes/Login/index.jsx

```diff
@@ -1,19 +1,84 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import TextField from '@material-ui/core/TextField';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import CardMedia from '@material-ui/core/CardMedia';
+
+import '../../styles/login.css';
+import logo from '../../assets/logo.png';
 
 const Login = () => (
-    <div className="grid-element">
-      <div>
-          This is the Admin Page of Hack Illinois 2019. This page is in progress.
-      </div>
-      <span className="login">
-          <div>
-              <input placeholder="User Name" />
-          </div>
-          <div>
-              <input placeholder="Password" />
-          </div>
-      </span>
-    </div>
+    <Card id="card">
+        {/* Title */}
+        <AppBar position="static" color="default">
+            <Toolbar>
```

Comment:
I think this could look better visually. Try experimenting with background color, centering the text, font color, and font weight.

Something to try might be a dark blue background, centered text, and white, bold font

Reply:
I just removed the app bar so that it looks simpler. Here's what it looks like now:

![admin-portal](https://user-images.githubusercontent.com/10408882/48473666-91293700-e7be-11e8-9fa1-8d36dc2b92f5.png)